explorer/src/main/java/com/scalar/client/tool/explorer/command/Validate.java

```diff
@@ -51,10 +52,18 @@
   public void run() {
     ExplorerExecutor executor =
         explorer -> {
+          StatusCode statusCode;
```

Comment:
Does it need  to be here ?

Reply:
Thank you for review comments.
Sorry, it's better to initialize inside the loop.
Updated review changes in fafec7963f9bf50fc4d58d6e8c3c956b9ef0458b